src/ORM/Table.php

```diff
@@ -1730,14 +1730,21 @@ protected function _processFindOrCreate($search, callable $callback = null, $opt
         }
         $entity = $this->newEntity();
         if ($options['defaults'] && is_array($search)) {
-            $entity->set($search, ['guard' => false]);
+            $accessibleFields = array_combine(array_keys($search), array_fill(0, count($search), true));
```

Comment:
I am not too happy about this.
This creates still a dangerous case where only the submitted fields are validated, right?
As such a half validated entity (some of the required fields as per validation are not in errors list) could be persisted?

Reply:
OK, never mind: The test case proofs that this is only about accessible fields, not about the validation fields on required at "create": https://github.com/cakephp/cakephp/pull/13066